spec/flipper/adapters/http_spec.rb

```diff
@@ -0,0 +1,399 @@
+require 'helper'
+require 'flipper/adapters/http'
+require 'webmock/rspec'
+require 'flipper/spec/shared_adapter_specs'
+
+RSpec.describe Flipper::Adapters::Http do
```

Comment:
yes that is for sure a great idea and have been thinking of some ways to do it.  I'll check out those links and find a way to make this happen because I think it is important to make sure its tested against the shared specs.